packages/edit-site/src/components/template-actions/rename-menu-item.js

```diff
@@ -64,14 +68,14 @@ export default function RenameMenuItem( { template, onClose } ) {
 
 	return (
 		<>
-			<MenuItem
+			<UsedComponent
 				onClick={ () => {
```

Comment:
I'm not sure if `onClick` works at this point on `DropdownMenuItemV2` — you may need to use `onSelect` instead.